wof/dao.py

```diff
@@ -1,127 +1,129 @@
 from __future__ import (absolute_import, division, print_function)
 
+msg = '{} Method not implemented by this service.'.format
```

Comment:
Ah okay. That's so cool. I will definitely look into that aspect of it further. Thanks for sharing your amazing python knowledge! 😄